core/src/main/java/io/smallrye/context/impl/ThreadContextProviderPlan.java

```diff
@@ -41,14 +41,14 @@ public ThreadContextProviderPlan(Set<ThreadContextProvider> propagatedSet, Set<T
         if (fast) {
             // true means check that all providers support it
             for (ThreadContextProvider provider : propagatedProvidersFastIterable) {
-                if (provider instanceof FastThreadContextProvider == false) {
+                if (!(provider instanceof FastThreadContextProvider)) {
```

Comment:
I really find that a lot less readable :(

Reply:
I think it's better, but I don't have a particularly strong opinion, so I'll revert